src/js/themes/base.d.ts

```diff
@@ -549,24 +549,88 @@ export interface ThemeType {
   button?: ButtonType;
   calendar?: {
     day?: {
+      hover?: {
+        background?: BackgroundType;
+        color?: ColorType;
+      };
+      selected?: {
+        background?: BackgroundType;
+        color?: ColorType;
+        font?: {
+          weight?: string | number;
+        };
+        hover?: {
+          background?: BackgroundType;
+          color?: ColorType;
+        };
+      };
+      inRange?: {
+        color?: ColorType;
+        font?: {
+          weight?: string | number;
```

Comment:
Im surprised we dont have this pulled out as `WeightType` but thats fine if we leave as is or pull it out

Reply:
Yeah, let's leave as is for now to minimize changes in a big PR.